packages/react/src/components/Link.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { BuilderStoreContext } from '../store/builder-store'
+
+export const Link: React.SFC<React.AnchorHTMLAttributes<
+  HTMLAnchorElement
+>> = props => (
```

Comment:
was this formatted by prettier like this? haven't seen wrapping like this before

Reply:
yes , I ran prettier on both changed files in this PR